src/history/history-and-comments.component.tsx

```diff
@@ -87,10 +89,11 @@ const HistoryAndComments: React.FC<{
                 {t("dispensedMedication", "dispensed medication")} -{" "}
                 {formatDatetime(parseDate(dispense.whenHandedOver))}
               </h5>
-              <MedicationEventCard
-                medicationEvent={dispense}
-                actionMenu={generateMedicationDispenseActionMenu(dispense)}
-              />
+              <Tile className={styles.dispenseTile}>
+                {generateMedicationDispenseActionMenu(dispense)}
+                <Tag type="red">{t("dispense", "Dispense")}</Tag>
+                <MedicationEvent medicationEvent={dispense} />
+              </Tile>
```

Comment:
This is a clearer example of the what changed in the MedicationEventCard.... the Tile now leaves here (so we can apply custom styling) as well the action menu, and the new "Dispense" tag... the Medication Event card just displays the details of the medication event.